--- /usr/src/astguiclient_snapshot_2006-04-10/VICIDIAL_web/admin.php	2006-02-27 15:58:50.000000000 -0800
+++ /var/www/html/vicidial/admin.php	2006-04-10 23:12:19.000000000 -0700
@@ -3320,15 +3320,38 @@
 	$o=0;
 	while ($statuses_to_print > $o) 
 	{
-		$rowx=mysql_fetch_row($rsltx);
-	$leads_in_list = ($leads_in_list + $rowx[2]);
+	    $rowx=mysql_fetch_row($rsltx);
+	    
+	    $lead_list['count'] = ($lead_list['count'] + $rowx[2]);
+	    if ($rowx[1] == 'N') 
+	    {
+		$since_reset = 'N';
+		$since_resetX = 'Y';
+	    }
+	    else 
+	    {
+		$since_reset = 'Y';
+		$since_resetX = 'N';
+	    } 
+	    $lead_list[$since_reset][$rowx[0]] = $rowx[2];
+	    $lead_list[$since_reset.'_count'] = ($lead_list[$since_reset.'_count'] + $rowx[2]);
+	    #If opposite side is not set, it may not in the future so give it a value of zero
+	    if (!isset($lead_list[$since_resetX][$rowx[0]])) 
+	    {
+		$lead_list[$since_resetX][$rowx[0]]=0;
+	    }
+	    $o++;
+	}
 
+	$o=0;
+	while (list($dispo,) = each($lead_list[$since_reset]))
+	{
 	if (eregi("1$|3$|5$|7$|9$", $o))
 		{$bgcolor='bgcolor="#B9CBFD"';} 
 	else
 		{$bgcolor='bgcolor="#9BB9FB"';}
 
-	if ($rowx[0] == 'CBHOLD')
+	if ($dispo == 'CBHOLD')
 		{
 		$CLB="<a href=\"$PHP_SELF?ADD=811&list_id=$list_id\">";
 		$CLE="</a>";
@@ -3338,24 +3361,12 @@
 		$CLB='';
 		$CLE='';
 		}
-
-	if ($rowx[1] == 'N')
-		{
-		$leads_in_list_N = ($leads_in_list_N + $rowx[2]);
-		echo "<tr $bgcolor><td><font size=1>$CLB$rowx[0]$CLE</td><td><font size=1> </td><td><font size=1>$rowx[2]</td></tr>\n";
-		}
-	else 
-		{
-		$leads_in_list_Y = ($leads_in_list_Y + $rowx[2]);
-		echo "<tr $bgcolor><td><font size=1>$CLB$rowx[0]$CLE</td><td><font size=1>$rowx[2]</td><td><font size=1> </td></tr>\n";
-
-		}
-
-		$o++;
+	echo "<tr $bgcolor><td><font size=1>$CLB$dispo$CLE</td><td><font size=1>".$lead_list['Y'][$dispo]."</td><td><font size=1>".$lead_list['N'][$dispo]." </td></tr>\n";
+	$o++;
 	}
 
-echo "<tr><td><font size=1>SUBTOTALS</td><td><font size=1>$leads_in_list_Y</td><td><font size=1>$leads_in_list_N</td></tr>\n";
-echo "<tr bgcolor=\"#9BB9FB\"><td><font size=1>TOTAL</td><td colspan=2 align=center><font size=1>$leads_in_list</td></tr>\n";
+echo "<tr><td><font size=1>SUBTOTALS</td><td><font size=1>$lead_list[Y_count]</td><td><font size=1>$lead_list[N_count]</td></tr>\n";
+echo "<tr bgcolor=\"#9BB9FB\"><td><font size=1>TOTAL</td><td colspan=2 align=center><font size=1>$lead_list[count]</td></tr>\n";
 
 echo "</table></center><br>\n";
 
