--- ./include/utils/UserInfoUtil.php	2008-12-31 21:39:39.000000000 -0500
+++ ./include/utils/UserInfoUtil.php	2009-01-01 01:20:17.000000000 -0500
@@ -1175,6 +1175,18 @@
 	global $adb;
 	global $current_user;
 	global $seclog;
+	if (!file_exists('user_privileges/user_privileges_'.$current_user->id.'.php'))
+        {
+                $up = fopen('user_privileges/user_privileges_'.$current_user->id.'.php',"a");
+                fwrite($up, '<? $placeholder=0; ?>');
+                fclose($up);
+        }
+	if (!file_exists('user_privileges/sharing_privileges_'.$current_user->id.'.php'))
+	{
+		$sp = fopen('user_privileges/sharing_privileges_'.$current_user->id.'.php',"a");
+		fwrite($sp, '<? $placeholder=0; ?>');
+		fclose($sp);
+	}
 	require('user_privileges/user_privileges_'.$current_user->id.'.php');
 	require('user_privileges/sharing_privileges_'.$current_user->id.'.php');
 	$permission = "no";
